TYPO3.Media/Migrations/Mysql/Version20130522131641.php

```diff
@@ -5,7 +5,7 @@
 use Doctrine\DBAL\Schema\Schema;
 
 /**
- * Adjust TYPO3.Media tables to new Asset base class and add new models.
+ * Adjust Neos.Media tables to new Asset base class and add new models.
```

Comment:
This is "historic material", please leave those old migrations alone, to keep their "inner consistency".